changelog page desktop & mobile

touchRED opened the pull request:
Closes #88, adding the changelog page which details updates to the SEDA data & explorer at the 4.0 release. 

Notes:
- adds partial for the data table svg located in the hero
- adds new page at `/whats-new/` route
- used my best judgement on the tablet & mobile breakpoints, lmk if there's anything you'd like me to change

lougroshek commented on content/whats-new/_index.md:
```diff
@@ -0,0 +1,7 @@
+type: "whats-new"
+is_index: true
+---
+Page added 1/18/21
```
I realize that this is an index page and this content in the .md body won't be rendered, but hugo has a default frontmatter date field for this purpose. I would recommend that you use that instead, and keep the .md body clean, just in case this page should be used in a different way in the future.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+    </div>
+  </div>
+</div>
+<div class="py-5 py-md-10 px-4 px-md-0" style="background-color: #F5F8FA;">
```
Why is there an inline style here? Please select these elements collectively and apply the background color in a stylesheet or define and apply a class instead.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+  </div>
+</div>
+<div class="py-5 py-md-10 px-4 px-md-0" style="background-color: #F5F8FA;">
+  <div style="max-width: 1440px; margin: 0 auto;">
```
Why is there an inline style here? Please select these elements collectively and apply the background color in a stylesheet or define and apply a class instead.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+  </div>
+</div>
+<div class="py-5 py-md-9 px-4 px-md-0">
+  <div style="max-width: 1440px; margin: 0 auto;">
```
Here is a repeat of your inline style from above. This page will be easier to maintain if you abstract these into classes or select a pattern for all of these elements for this page/page type.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+  </div>
+</div>
+<div class="py-5 py-md-9 px-4 px-md-0" style="background-color: #F5F8FA;">
+  <div style="max-width: 1440px; margin: 0 auto;">
```
Here is a repeat of your inline style from above. This page will be easier to maintain if you abstract these into classes or select a pattern for all of these elements for this page/page type.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+    </div>
+  </div>
+</div>
+<div class="py-5 py-md-9 px-4 px-md-0" style="background-color: #F5F8FA;">
```
Here is a repeat of your inline style from above. This page will be easier to maintain if you abstract these into classes or select a pattern for all of these elements for this page/page type.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+  </div>
+</div>
+<div class="py-5 py-md-9 px-4 px-md-0">
+  <div style="max-width: 1440px; margin: 0 auto;">
```
Here is a repeat of your inline style from above. This page will be easier to maintain if you abstract these into classes or select a pattern for all of these elements for this page/page type.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+    </div>
+  </div>
+</div>
+<div class="row map-invite map-invite-4 m-0 px-4 py-6 py-lg-9 py-xl-10" style="background-color: #F5F8FA;">
```
Here is a repeat of your inline style from above. This page will be easier to maintain if you abstract these into classes or select a pattern for all of these elements for this page/page type.

lougroshek commented:
@touchRED Looks good. partial is a great way to add that svg.

lougroshek commented on themes/base-hugo-theme/layouts/whats-new/list.html:
```diff
@@ -0,0 +1,100 @@
+    <p class="mb-4 px-0 col col-md-4 mx-auto text-center">We’ve refined the location data panel to provide a clearer view of statistics at a glance, and clicking any variable in the panel now updates the map and charts.  You can also compare several locations’ data together in a revised table display.</p>
+    <div class="d-flex flex-column flex-md-row justify-content-center align-items-center" >
+      <img class="col col-md-4 px-0 mb-4 mb-md-0 mr-0 mr-md-8" src="/images/new-layout.png" />
+      <img class="col col-md-4 px-0 mb-4 mb-md-0" src="/images/new-layout.png" />
```
Are these two images supposed to be the same? Maybe just an oversight?

touchRED commented:
thanks for the review @lougroshek! changes:
- moved inline styles into `whats-new.scss`
- the `date` frontmatter on `content/whats-new/_index.md` was already there, so I just removed the body content

As for the placeholder images, I expected to receive the proper images from design but haven't quite yet – no viable images to be found anywhere in figma either. There's a note about this next to one of the figma frames:

<img width="341" alt="Screen Shot 2021-01-20 at 10 28 43 PM" src="https://user-images.githubusercontent.com/1114157/105276984-8756a780-5b70-11eb-8579-369d6ff5e163.png">

This is obviously a very quick change, so if we can get those in the morning I'd be happy to get them situated in time for the demo!

lougroshek commented:
@touchRED Thanks for these changes.
